using-mobile-extensions/mobile-core/rules-engine/rules-engine-details.md

```diff
@@ -194,28 +199,25 @@ Here is the example:
 
 ## Consequence object definition
 
-The consequences section of a rule lists the file names of each consequence object that should be performed when the condition for that rule evaluates to `true`.
+The consequences section of a rule lists the file names of each consequence object that should be performed when all of the conditions for that rule evaluate to `true`.
 
-| **Friendly Name** | **Key    Value** | **Type** | **Description** |
+| **Friendly name** | **Key** | **Type** | **Description** |
 | :--- | :--- | :--- | :--- |
-| Identifier | `id` | string | String that contains a unique identifier for this consequence.  `sha1`, or another guaranteed random value with a near-impossible chance of collisions, is recommended. |
-| Consequence Type | `type` | string | A Consequence Type from the [Consequences Type](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine#consequence-types) table. |
-| Consequence Details | `detail` | object | JSON object that contains the details that are necessary to perform a consequence of the given type. |
+| Identifier | `id` | string | *(Required)* String that contains a unique identifier for this consequence.  `sha1`, or another guaranteed random value with a near-impossible chance of collisions, is recommended. |
+| Consequence type | `type` | string | *(Required)* A Consequence Type from the [Consequences Type](https://aep-sdks.gitbook.io/docs/using-mobile-extensions/mobile-core/rules-engine#consequence-types) table. |
+| Consequence details | `detail` | object | *(Required)* JSON object that contains the details that are necessary to perform a consequence of the given type. |
 
 ## Consequence types
 
 | **Name** | **Value** | **Description** | **Payload Definition** |
 | :--- | :--- | :--- | :--- |
-| Analytics | `an` | Sends data to Analytics |  |
-| In-App Message | `iam` | In-App Message | In-App Consequence Detail Definition |
-| Postback | `pb` | Send Postback\(s\) | Postback Consequence Detail Definition |
-| PII | `pii` | Sync PII | Sync PII Consequence Detail Definition |
-| Open URL | `url` | Passes the provided URL to be opened by the platform that is most commonly used for app deeplinking. | Open URL Consequence Detail Definition |
-| Callback | `cb` | Calls a customer-defined method. |  |
-| Target | `tar` | Send a request to Target with location information to enrich the user's profile. |  |
-| Audience Manager | `aam` | Send a request to Audience Manager with location information to enrich the user's profile. |  |
-| Client Side Profile | `csp` | Create or delete operations against the client-side profile. | Profile Consequence Detail Definition |
-| Generate Event | `event` | Generate a raw event on the event hub. |  |
+| Analytics | `an` | Sends data to Analytics | [Analytics consequence detail definition](rules-engine-consequence-details.md#analytics-consequence) |
```

Comment:
please add a period after the bracket with the URL for all of the "description" sentences in this table.